images/dockerregistry/config.yml

```diff
@@ -35,3 +35,8 @@ middleware:
         blobrepositorycachettl: 10m
   storage:
     - name: openshift
+openshift:
+  version: 1.0
+  metrics:
+    enabled: false
+    secret: TopSecretToken
```

Comment:
It should be impossible for a user to accidentally turn this on and have the default secret in place.